web/func_equal_test.go

```diff
@@ -65,7 +65,7 @@ func TestFuncEqual(t *testing.T) {
 		r := funcEqual(test.a, test.b)
 		if r != test.result {
 			t.Errorf("funcEqual(%v, %v) should have been %v",
-				test.a, test.b, test.result)
+				test.a(), test.b(), test.result)
```

Comment:
The idea behind this test was to print the function values, not their return values. What was the error that caused you to make this change?

Reply:
go1.10 test's print this error.

```
# github.com/zenazn/goji/web
web/func_equal_test.go:67: Errorf format %v arg test.a is a func value, not called
```

but fmt package can print.
https://play.golang.org/p/r7HMC0wyydX

Reply:
this is strange. specified function test is succeeded.

__failed__

```bash
$ go test -v .
# github.com/smith-30/goji/web
./func_equal_test.go:67: Errorf format %v arg test.a is a func value, not called
FAIL    github.com/smith-30/goji/web [build failed]
```

__succeeded__

```
Running tool: /usr/local/bin/go test -v -timeout 15s github.com/smith-30/goji/web -run ^TestFuncEqual$

=== RUN   TestFuncEqual
--- PASS: TestFuncEqual (0.00s)
PASS
ok  	github.com/smith-30/goji/web	0.014s
Success: Tests passed.
```

Reply:
I think that the result will not change with this, but how about it?

```
for _, test := range funcEqualTests {
		r := funcEqual(test.a, test.b)
		if r != test.result {
			aAddr := reflect.ValueOf(&test.a).Elem()
			bAddr := reflect.ValueOf(&test.b).Elem()
			t.Errorf("funcEqual(%v, %v) should have been %v",
				aAddr, bAddr, test.result)
		}
	}
```